File: plugins/flutter_bg_location_plugin/ios/Classes/Handler/LocationServiceMetaHandler.swift
import  Flutter

final class LocationServiceMetaHandler: Handler {

    let callMethod = CallMethods.LOCATION_SERVICE_META

    func handler(call: FlutterMethodCall, result: @escaping FlutterResult) {

        let storage = PluginContext.shared.locationStorage;
        let meta: [String: Any] = [
            "tickerSeconds": storage.getTickerSeconds(),
            "tickersCount":  storage.getTickers(),
            "hash":          storage.getHash(),
            "orderId":       storage.getOrderId()
        ]    
        result(meta)
    }
}

File: plugins/flutter_bg_location_plugin/android/src/main/kotlin/com/ezride/flutter_bg_location_plugin/handlers/LocationServiceMetaHandler.kt
package com.ezride.flutter_bg_location_plugin.handlers

import io.flutter.plugin.common.MethodCall
import io.flutter.plugin.common.MethodChannel
import android.content.Context
import android.content.Intent
import android.util.Log
import com.ezride.flutter_bg_location_plugin.services.LocationService
import com.ezride.flutter_bg_location_plugin.models.LocationServiceMeta

class LocationServiceMetaHandler : Handler{


    override val callMethod : String = CallMethods.LOCATION_SERVICE_META;

    override fun handler(context: Context, call: MethodCall, result: MethodChannel.Result){
        val meta :LocationServiceMeta = LocationService.getMeta(context)
        val map = mapOf(
            "tickerSeconds" to meta.tickerSeconds,
            "tickersCount"   to meta.tickersCount,
            "hash"          to meta.hash,
            "orderId"       to meta.orderId
        )
        result.success(map)
    }
}